### DISCUSSION TITLE
I can't set up a new rails project with tailwind-css

### DISCUSSION THREAD

**Discussion by deyemiobaa** [tailwindlabs/tailwindcss#8915] [Help] [answered]
Hello, I'm trying to set up a new rails project with `rails new my_app --css tailwind` && `rails new my_app --css=tailwind` but I keep getting an error
![image](https://user-images.githubusercontent.com/55185309/180105856-9ccf9863-e711-4f00-af84-12420a0110f1.png)

This same error also exists if I try to add tailwind-CSS manually using the steps specified in the documentation
![image](https://user-images.githubusercontent.com/55185309/180106000-913b8f61-3599-499e-aebb-47fe4b527e46.png)

I use a Windows 10 PC

**Comment by benkoshy** [comment, marked as answer]
are you able to manually do it with this gem: https://github.com/rails/cssbundling-rails ?

**Reply by deyemiobaa** [reply]
This worked perfectly! Thank you!

**Reply by deyemiobaa** [reply]
Hi, @benkoshy. So I'm currently faced with a different issue. The CSS in assests/builds/application.css doesn't get added to the bundled code. And that prevents the style from reflecting on the webpage

Any fix for this?

**Reply by benkoshy** [reply]
Very very hard to debug like this.

If you're working on Windows, be prepared to hit a lot of little road blocks along the way as well.

If it's working in development, but not production -- i'm not sure. if it's not working in development, ensure you have the appropriate stylesheet link tag (I forget the exact name) and also ensure that there is no conflict with sass application.css as well.

you really need to provide a github repositiory to debug, otherwise it's impossible.

**Reply by deyemiobaa** [reply]
@benkoshy this is a link to the repo https://github.com/deyemiobaa/blog_app/tree/feature/forms

I've pushed all the changes. contents of app/builds were added to .gitignore by default but I disabled that so you could see the output

**Reply by benkoshy** [reply]
don't disable.................they will be created by the asset pipeline on the fly.

**Reply by deyemiobaa** [reply]
@benkoshy just pushed. reverted it

**Comment by benkoshy** [comment]
You might wanna post on a rails forum - someone should answer
